tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -212,6 +283,10 @@ def is_video_play_button_visible(self):
     def is_progress_bar_showing(self):
         return self.is_element_displayed(*self._video_progress_bar_locator)
 
+    @property
+    def not_video_preview(self):
+	return self.is_element_displayed(*self._not_video_preview_locator)
+
```

Comment:
I ran this, and this won't work to simply detect the image preview.  If you view the app via WebIDE, in case of video preview, following is displayed: <img class="videoPoster" style="display: block...
But in case of image preview, following is displayed: <img class="image-view" src="blob:app://camera.gaiamoble.org...
So your method should check for the visibility of those classes to see whether it's an image view or video view.